.github/workflows/go.yaml

```diff
@@ -29,6 +29,46 @@ jobs:
         only-new-issues: true
         args: --timeout=2m
 
+  build-controller:
+    name: test
+    runs-on: ubuntu-latest
```

Comment:
:astonished: no fedora/centos targets?

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners
No options for that